src/screens/EWayBill/ListEWayBillsScreen.tsx

```
@@ -0,0 +1,358 @@
+        } catch (error) {
+            console.log("Error while fetching tax numbers", error);
+            Toast({ message: error?.message, duration: 'SHORT', position: 'BOTTOM' })
+        }
+    }
+
+    const fetchEWayBills = async (page?: number) => {
+        try {
+            const _page = page ?? pageData.page + 1;
+            const response = await CommonService.fetchEWayBills(date.startDate, date.endDate, selectedGst, _page);
+            if (response && response?.status == "success") {
+                const { results, page, totalPages } = response?.body;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)